currency/pairs.go

```diff
@@ -199,28 +199,25 @@ func (p Pairs) GetPairsByCurrencies(currencies Currencies) Pairs {
 	return pairs
 }
 
-// Remove removes the specified pair from the list of pairs if it exists
-func (p Pairs) Remove(pair Pair) (Pairs, error) {
-	pairs := slices.Clone(p)
-	for x := range p {
-		if p[x].Equal(pair) {
-			return append(pairs[:x], pairs[x+1:]...), nil
+// Remove removes the specified pairs from the list of pairs if they exist
+func (p Pairs) Remove(rem ...Pair) Pairs {
+	var n Pairs
```

Comment:
`n := make(Pairs, 0, len(p))`

Reply:
I don't tend to pre-alloc if we don't know the size, but yeah, that's fine.
I can always `Shrink`

Reply:
Fixed 99967da